src/Compilers/CSharp/CscCore/csc.deps.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
That file though looks completely machine read / written only.  Don't think any humans are going to be messing with that.  That makes me not sure how we manage this file going forward.